android/SherpaOnnx/app/src/main/java/com/k2fsa/sherpa/onnx/SherpaOnnx.kt

```diff
@@ -85,14 +85,17 @@ class SherpaOnnx(
         acceptWaveform(ptr, samples, sampleRate)
 
     fun inputFinished() = inputFinished(ptr)
-    fun reset(recreate: Boolean = false) = reset(ptr, recreate = recreate)
+    fun reset(recreate: Boolean = false, hotWords: String = "") = reset(ptr, recreate, hotWords)
```

Comment:
Could you also update
https://github.com/k2-fsa/sherpa-onnx/blob/558f5e32630e056f776ea2c3f3c3b1c062748410/android/SherpaOnnx2Pass/app/src/main/java/com/k2fsa/sherpa/onnx/SherpaOnnx.kt#L123